Libs/CodeGen/ApiGen/src/main/resources/api/doc.api

```diff
@@ -114,6 +114,10 @@ api(Doc) {
     @entitle=/data/write/$f(docUri)
     @public List(String) deleteDocsByUriPrefix(String docUri);
    
+    [Removes a folder and its contents recursively, including empty subfolders. Validates entitlement on individual docs and folders. Returns a list of the documents and folders removed.]
+    @entitle=/data/write/$f(docUri)
+    @public List(String) deleteDocsByUriPrefix2(String docUri, Boolean recurse);
```

Comment:
Since this function only makes a difference if the boolean value is true (vs the original deleteDocsByUriPrefix function), I vote to not have 2 separate functions and to just have 1 function that takes both arguments

deleteDocsByUriPrefix(String uri, Boolean recurse)

We should then standardize this across the other APIs for the delete*ByUriPrefix functions.

I really don't like the function just being called deleteDocsByUriPrefix2 though.  Thoughts @jonathan-major, @scarabus, @amkimian